testsuite/model/src/test/java/org/keycloak/testsuite/model/KeycloakModelTest.java

```diff
@@ -637,4 +640,25 @@ protected static RealmModel createRealm(KeycloakSession s, String name) {
         return realm;
     }
 
+    /**
+     * Moves time on the Keycloak server as well as on the remote Infinispan server if the Infinispan is used.
+     * The caller of the method is responsible to reset time offset (by calling advance(0)) after a test run.
+     * @param seconds time offset in seconds by which Keycloak (and Infinispan) server time is moved
+     */
+    protected void advanceTime(int seconds) {
```

Comment:
Can we change the name to `setTimeOffset` to be consistent with the base testsuite?

Reply:
done